.changeset/sour-dingos-rush.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
**Suggestion:**
I think we should have several changesets here
* One for `alfa-result`, which is breaking (should be major, down to minor because of pre-release). This should focus on the API change and advice (as quick migration fix) to replace `foo.get()` by `foo.getUnsafe()`.
* One for `alfa-web`. This is, technically, a different breaking change where the `from` functions now return `Result` instead of the object. This would advice to add `.getUnsafe()` call as a quick migration fix.
* Maybe one for all the rest. I'm not fully convinced it is needed since these do not have any API or functional change, so there is nothing that really need to be conveyed to consumers.

Splitting the changesets means that each packages CHANGELOG will only contain its own changes. That is, consumers of `alfa-result` don't care about the change in `alfa-web`, so the `alfa-result` changelog shouldn't list them.